src/scripts/bee.js

```diff
@@ -143,7 +143,12 @@ class Bee extends MovingObject {
   }
 
   setTrajectory(direction) {
-    const nudgeFactor = direction === "up" ? CONSTANTS.NUDGE : -CONSTANTS.NUDGE;
+    const nudgeFactor =
+      direction === "up"
+        ? CONSTANTS.NUDGE
+        : direction === "down"
+        ? -CONSTANTS.NUDGE
+        : 0;
```

Comment:
This chained ternary is hard to read and will be easy to have bugs if someone changes it in the future.

It's better to be more verbose in this case, either w/ a switch or if/else blocks.

Reply:
Same comment in other places.

Reply:
For sure, refactoring to use switch statements for better readability.